src/utils/fileUtils.ts

```diff
@@ -0,0 +1,76 @@
+import * as path from "path";
+import { TextDocument, window, workspace, WorkspaceFolder } from "vscode";
+
+/**
+ * Return the current workspace uri from root
+ *
+ * @param currentFile the opened TextDocument
+ * @returns the path from root of the current workspace
+ */
+export function getWorkspaceUri(currentFile: TextDocument): string {
+  const currentWorkspace: WorkspaceFolder = (currentFile && currentFile.uri) ?  workspace.getWorkspaceFolder(currentFile.uri) : undefined;
+ return ((currentWorkspace && currentWorkspace.uri.fsPath) || (workspace.workspaceFolders && workspace.workspaceFolders[0].uri.fsPath));
+}
+
+/**
+ * Return the current workspace absolute uri
+ *
+ * @param currentFile the opened TextDocument
+ * @returns the absolute path of the current workspace
+ */
+export function getAbsoluteWorkspaceUri(currentFile: TextDocument): string {
+  const currentWorkspace: WorkspaceFolder = (currentFile && currentFile.uri) ?  workspace.getWorkspaceFolder(currentFile.uri) : undefined;
+  return ((currentWorkspace && currentWorkspace.uri) || (workspace.workspaceFolders && workspace.workspaceFolders[0].uri)).toString();
+}
+
+/**
+ * Retrieve the text document that is open in the editor
+ *
+ * @returns the text document open in the editor
+ */
+export function getCurrentFile(): TextDocument {
+  return (window.activeTextEditor && window.activeTextEditor.document && window.activeTextEditor.document.languageId === 'xml') ?  window.activeTextEditor.document : undefined;
+}
+
+/**
+ * Return the uri of the current file
+ *
+ * @param currentFile the opened TextDocument
+ * @returns the uri of the current file
+ */
+export function getCurrentFileUri(currentFile: TextDocument): string {
+  return (currentFile && currentFile.uri) ? currentFile.uri.fsPath : undefined;
+}
+
+/**
+ * Uses path to return a basename from a uri
+ *
+ * @param filePath the uri of the file
+ * @return the filename
+ */
+export function getFileName(filePath: string): string {
+  return path.basename(filePath);
+}
+
+/**
+ * Return the relative file path between a start and destination uri
+ *
+ * @param startPath the absolute path of the beginning directory
+ * @param destinationPath the absolute path of destination file
+ * @returns the path to the destination relative to the start
+ */
+export function getRelativeUri(startPath: string, destinationPath: string): string {
+  const relative = path.relative(path.normalize(startPath), path.normalize(destinationPath));
```

Comment:
Rename with getRelativePath ans rename directly the value

Reply:
Sorry, forgot to remove after debug